packages/advanced-logic/specs/payment-network-any-declarative-0.1.0.md

```diff
@@ -26,13 +18,22 @@ As a payment network, this extension allows to deduce a payment `balance` for th
 | **version**                      | String | constant value: "0.1.0"                        | **Mandatory** |
 | **events**                       | Array  | List of the actions performed by the extension | **Mandatory** |
 | **values**                       | Object |                                                |               |
-| **values.sentPaymentAmount**     | Amount | Amount of payment declared sent                | **Mandatory** |
-| **values.sentRefundAmount**      | Amount | Amount of refund declared sent                 | **Mandatory** |
-| **values.receivedPaymentAmount** | Amount | Amount of payment declared received            | **Mandatory** |
-| **values.receivedRefundAmount**  | Amount | Amount of refund declared received             | **Mandatory** |
+| **values.sentPaymentAmount**     | Amount | Total amount of payments declared sent                | **Mandatory** |
+| **values.sentRefundAmount**      | Amount | Total amount of refunds declared sent                 | **Mandatory** |
+| **values.receivedPaymentAmount** | Amount | Total amount of payments declared received            | **Mandatory** |
+| **values.receivedRefundAmount**  | Amount | Total amount of refunds declared received             | **Mandatory** |
 | **values.paymentInstruction**    | String | Instruction to make payments                   | Optional      |
 | **values.refundInstruction**     | String | Instruction to make refunds                    | Optional      |
 
+## Interpretation
+
+Payments are described by `receivedPaymentAmount`.
```

Comment:
Are we getting rid of the balance?

Reply:
Yes because the balance can be explained once and for all, in my opinion (in payment-networks). It avoids a lot of repetitions.

Reply:
The file was missing, my bad. Please check again !